index.js

```diff
@@ -0,0 +1,134 @@
+document.addEventListener("DOMContentLoaded", () => {
+    let start = document.querySelector("#start");
+    let deck_id;
+    let score = 0;
+    let dscore = 0;
+    let imgArr = []
+    let playerScore;
+    let dealerScore;
+    let game = document.querySelector("#game")
+    let hitBtn = document.querySelector("#hit")
+    let stayBtn = document.querySelector("#stay")
+    let begin = document.querySelector("#begin")
+    let player = document.querySelector("#player")
+    let dealer = document.querySelector("#dealer")
+    let displayResults = document.createElement("h1")
+   
+    const drawCards = async () => {
+        try {
+            let res = await axios.get("https://deckofcardsapi.com/api/deck/new/");
+            deck_id = res.data.deck_id
+            let shuffled = await axios.get(`https://deckofcardsapi.com/api/deck/${deck_id}/shuffle/`);
+            let draw = await axios.get(`https://deckofcardsapi.com/api/deck/${deck_id}/draw/?count=2`);
+
+            for(let i = 0; i < draw.data.cards.length; i++){
+                let playerImg = document.createElement("img");
+                let src = draw.data.cards[i].image;
+                playerImg.src = src;
+                player.appendChild(playerImg);
+            }
+
+            playerScore = document.createElement("h3")
+            for(let j = 0; j < draw.data.cards.length; j++){
```

Comment:
It would be a good idea to assign draw.data.cards to a variable with a short name and use that variable in your for loop since you are using it 6 times.